daisy_rani_kesavan/week_01/day05/subway.html

```diff
@@ -0,0 +1,65 @@
+<html>
+<head>
+	Subway program
+</head>
+<body>
+<script type="text/javascript">
+
+var N=['Times Square','34th','28th','23rd','Union Square', '8th'];
+var L=['8th','6th','Union Square','3rd','1st'];
+var six=['Grand Central','33rd','28th','23rd','Union Square','Astor'];
+
+function rev(line1, stop1, line2, stop2){
```

Comment:
Consider renaming this function to something a little more helpful to me, trying to understand what it does. I usually like 2 word method names with 1 verb and 1 noun. `parkCar`, `planTrip`, etc.